### ISSUE TITLE
1.10 - Some disguises crash the client.

### ISSUE THREAD

**Issue by mibby** [libraryaddict/LibsDisguises#144]
LibsDisguise dev 36.

Had reports from a few players that some disguises crash their client and anyone nearby. The few that have been reported to me thus far are sheep and baby horse disguises. Occurs when view self disguise is enabled. I assume it would crash others as well when disguising as a mob if self disguise is disabled since it crashes others with it enabled.

```
Description: Ticking entity

java.lang.ClassCastException: java.lang.Byte cannot be cast to java.lang.Boolean
	at rt.l(SourceFile:71)
	at wc.n(SourceFile:43)
	at wl.n(SourceFile:122)
	at sf.m(SourceFile:1654)
	at sg.m(SourceFile:277)
	at aid.a(SourceFile:1520)
	at aid.g(SourceFile:1498)
	at aid.k(SourceFile:1388)
	at bcx.t(SourceFile:1652)
	at bcx.av(SourceFile:968)
	at bcx.a(SourceFile:396)
	at net.minecraft.client.main.Main.main(SourceFile:124)

-- Head --
Thread: Client thread
Stacktrace:
	at rt.l(SourceFile:71)
	at wc.n(SourceFile:43)
	at wl.n(SourceFile:122)
	at sf.m(SourceFile:1654)
	at sg.m(SourceFile:277)
	at aid.a(SourceFile:1520)
	at aid.g(SourceFile:1498)

-- Entity being ticked --
Details:
	Entity Type: Sheep (wl)
	Entity ID: -10
	Entity's Vehicle: ~~ERROR~~ NullPointerException: null
Stacktrace:
	at aid.k(SourceFile:1388)
```

**Comment by mibby** [comment]
```
Description: Rendering entity in world

java.lang.ClassCastException: java.lang.Byte cannot be cast to java.lang.Boolean
	at rt.l(SourceFile:71)
	at rt.l_(SourceFile:163)
	at bsy.a(SourceFile:90)
	at btb.a(SourceFile:41)
	at btb.a(SourceFile:16)
	at bsh.a(SourceFile:342)
	at bsh.a(SourceFile:329)
	at boh.a(SourceFile:603)
	at bnz.a(SourceFile:1266)
	at bnz.b(SourceFile:1171)
	at bnz.a(SourceFile:1018)
	at bcx.av(SourceFile:991)
	at bcx.a(SourceFile:396)
	at net.minecraft.client.main.Main.main(SourceFile:124)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at rt.l(SourceFile:71)
	at rt.l_(SourceFile:163)
	at bsy.a(SourceFile:90)
	at btb.a(SourceFile:41)
	at btb.a(SourceFile:16)

-- Entity being rendered --
Details:
	Entity Type: Sheep (wl)
	Entity ID: -10
```

**Comment by mixxit** [comment]
Also getting this

http://pastebin.com/4EW192Q5

**Comment by mixxit** [comment]
Confirmed as fixed for blazes in build #39
